apps/src/calc/expressionNode.js

```diff
@@ -115,6 +125,7 @@ ExpressionNode.prototype.evaluate = function (globalMapping, localMapping) {
     localMapping = localMapping || {};
 
     var type = this.getType_();
+    var val;
```

Comment:
Maybe JSDoc style type hint here, that it might be a `@type {number|JSNum}` since jsnum sometimes returns numbers, sometimes jsnumbers